src/groups/bmq/bmqt/bmqt_uri.t.cpp

```diff
@@ -657,6 +657,28 @@ static void test7_testLongUri()
     bmqt::UriParser::shutdown();
 }
 
+static void test8_testUriParserOverflow()
+{
+    s_ignoreCheckDefAlloc = true;
+    // Disable the default allocator check. When 'bmqt::Uri'
+    // is created from a long uri string error message is
+    // logged via 'BALL_LOG_ERROR' which allocates using
+    // the default allocator.
+
+    mwctst::TestHelper::printTestName("URI PARSER OVERFLOW TEST");
+
+    // Initialize bmqt::UriParser many times, and then deinitialize
+    // it the same number of times according to the usage contract.
+    const bsls::Types::Int64 k_INIT_NUM = 2200000000LL;
+    for (size_t i = 0; i < k_INIT_NUM; i++) {
```

Comment:
Does this give a signed/unsigned comparison warning?  I think picking either `size_t` or `Int64` is the right path here.

Reply:
Oh nvm a later commit did exactly this change.